### PR TITLE
CONCF-543 check if article exists and only then add it to API response

### PR DESCRIPTION
**Pull request by bognix**
@vforge @rogatty

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php [hunk 1]
```diff
@@ -263,27 +263,37 @@ public function processCuratedContent( $data ) {
...
 	 * Platform Team is OK with hitting DB for MVP (10-15 wikis)
 	 *
 	 * @param $item
 	 * @return mixed
 	 */
-	private function addArticleUrlToCuratedContentItem( $item ) {
+	private function processCuratedContentItem( $item ) {
 		if ( !empty( $item[ 'article_id' ] ) ) {
-			$item[ 'article_local_url' ] = Title::newFromID( $item[ 'article_id' ] )->getLocalURL();
+			$title = Title::newFromID( $item[ 'article_id' ] );
```
Comment (hakubo) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: hmm that is a call to DB for every article,
do we have to check it?
maybe we can handle nulls on the front end?
or maybe we can optimize it?

**Comment by hakubo** [comment]
LGTM

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php (hunk 1 above)
Comment (bognix) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: there is a comment above to check with platform team when we would go 100% or even for phase 2.

Comment (rogatty) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: Yup, it's stated in the method description that we should fix it.

Comment (vforge) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: We've talked about that with @owend and it's cool for Phase 1

**Comment by vforge** [comment]
LGTM

**Pull request merged**